mi/dataset/driver/pco2a_a/sample/pco2a_a_sample_driver.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+"""
+@package mi.dataset.driver.pco2a_a.sample.pco2a_a_sample_driver
+@file mi/dataset/driver/pco2a_a/sample/pco2a_a_sample_driver.py
+@author Tim Fisher
+@brief For creating a pco2a_a_sample driver.
+
+"""
+
+
+from mi.core.log import get_logger
```

Comment:
`get_logger` is outdated. Instead of:
```python
from mi.core.log import get_logger
log = get_logger()
```
use:
```python
from mi.core.log import log
```

Reply:
Done